authy/user.go

```diff
@@ -0,0 +1,111 @@
+package authy
```

Comment:
> I would suggest to move the packages `authy` and `wallety` and `txClient` under package `database` or even a folder under it, as this 3 modules are only useful for the simulation+database and nothing else, can you forseen other usages? Otherwise gaving them on the first nesting level it would confuse us.

i will move the TxClient inside as it's tight coupled to eth as we discussed. We keep wallet and authy outside for broader usage.